Bookish.DataAccess/CreateBooksTable.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE dbo.Books (
+	ISBN char(13) PRIMARY KEY NOT NULL,
+	Title varchar(50) NOT NULL,
+	Author varchar(50) NOT NULL,
+	TotalCopies int NOT NULL,
+	AvailableCopies int NOT NULL
```

Comment:
Hm, this doesn't help you keep track of which copies are available and which ones are on loan with which users. Having a separate BookCopies table or similar can probably help with that 🙂